merlin-certloader.rb

```diff
@@ -71,7 +71,7 @@ def fetch_release_metadata
     end
   end
 
-  @@version = "0.3.2"
```

Comment:
The new certloader version fixes this issue https://github.com/Shopify/certloader/pull/287